packages/server/lib/webhook/webhook.manager.ts

```diff
@@ -34,9 +34,17 @@ async function execute(
         return;
     }
 
+    const template = configService.getTemplate(integration.provider);
+
     const { environment, account } = environmentAndAccountLookup;
 
-    const handler = handlers[`${integration.provider.replace(/-/g, '')}Webhook`];
+    if (!template || !template['webhook_routing_script']) {
+        return;
+    }
+
+    const webhookRoutingScript = template['webhook_routing_script'];
+    const handler = handlers[webhookRoutingScript.replace(/-/g, '')];
```

Comment:
I am personally not super fan of this kind of routing based on filename/string. It makes it very hard to navigate/grep in an IDE. Could the keys in handlers be set explicitly to at least match the providers.yaml value?

Reply:
Fair, updated with https://github.com/NangoHQ/nango/pull/2516/commits/77155c4bb1a08d2832a6397844765aa132579c46